crates/dojo/world/src/local/artifact_to_local.rs

```diff
@@ -203,12 +203,12 @@ enum ResourceType {
 fn identify_resource_type(implem: &AbiImpl) -> ResourceType {
     if implem.interface_name == WORLD_INTF {
         ResourceType::World
-    } else if implem.interface_name == CONTRACT_INTF {
-        ResourceType::Contract(name_from_impl(&implem.name))
     } else if implem.interface_name == MODEL_INTF {
         ResourceType::Model(name_from_impl(&implem.name))
     } else if implem.interface_name == EVENT_INTF {
         ResourceType::Event(name_from_impl(&implem.name))
+    } else if implem.interface_name == DEPLOYED_RESOURCE_INTF {
+        ResourceType::Contract(name_from_impl(&implem.name))
```

Comment:
I'm a bit worried about the contract being a fallback, since models and events are also implementing this interface.

wdyt about adding a `IContract` again to ensure we have something well defined for contracts, even if for now the trait may contain only one function?
I'm not sure traits impl could be empty.

And if in the future we have some more characteristics for contracts, the trait is already there.

Reply:
Yes, let's use an empty `IContract` trait/impl 👍